numpy/_core/tests/test_regression.py

```diff
@@ -2636,3 +2636,11 @@ def test_repeated_square_consistency(self):
                 res = buf[3:]
                 np.square(in_vec, out=res)
                 assert_equal(res, expected_res)
+
+    def test_mult_scalar_vec_consistency(self):
+        # gh-26740 (the part about numpy scalar and vector inconsistency)
+        buf = np.array([-5.171866611150749e-07 + 2.5618634555957426e-07j,
+                        0, 0])
+        res0 = buf * buf
+        res1 = np.asarray([x * x for x in buf])
+        assert_equal(res0, res1)
```

Comment:
Use `assert_max_ulp` or `assert_almost_equal`. I think that should fix the CI failure.

Reply:
It fixes the CI failure, but I was aiming to solve the issue of scalar and array results differing at all (see #26740), and the failure is a real indicator that using `fma` for the scalar version does not fix the issue on all platforms.

I still think that the change is worthwhile because it increases scalar operations precision on some platforms (and makes it consistent with vector ops) but perhaps the test does not contribute if it would never fail without the fix?

Reply:
On the other hand, verifying that the scalar code stays consistent with the vectorized loop is good to do, so I kept the test with you suggestion to use `assert_almost_equal`.